webapp/channels/src/packages/mattermost-redux/src/reducers/entities/posts.ts

```diff
@@ -194,6 +194,7 @@ export function handlePosts(state: IDMappedObjects<Post> = {}, action: AnyAction
             [post.id]: {
                 ...state[post.id],
                 state: Posts.POST_DELETED,
+                message: '',
```

Comment:
These changes seem unrelated. Are they needed for the PR?

Reply:
I didn't think there was a need to persist the message in redux state so I just removed it